src/azure-stac-cli/stac/common/__pypgstac.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import subprocess
+from pathlib import Path
+
+def load_item(file_path: str):
+    """ Load stac item to PostgreSQL
+    :param file_path: Absolute path to the JSON file that contains the STAC Item
+    :type file_path: str
+    :returns: None
+    :rtype: None
+    """
+    
+    try:
+        cmd = f"pypgstac load items {file_path} --method insert".split()
+        
+        subprocess.check_call(cmd)
```

Comment:
I didn't see anything in their docs, but pypgstac is a python library. It feels like there should be a way to call their library vs having to fork a process that's just going to run python again.

Reply:
I agree. This was one of the feedback items that I provided on the original code as well. As I'm bringing in the code as-is for this PR, I will create a task for this change to be addressed in another PR.